slack.json

```diff
@@ -0,0 +1,5 @@
+{
+  "hooks": {
+    "get-hooks": "deno run -q --unstable --allow-read --allow-net https://deno.land/x/deno_slack_hooks@0.0.2/mod.ts"
```

Comment:
`0.0.1` is the latest release for the hooks lib, but I did just need to bump the version of the builder